textdb/textdb-common/src/main/java/edu/uci/ics/textdb/common/field/DoubleField.java

```diff
@@ -0,0 +1,47 @@
+package edu.uci.ics.textdb.common.field;
+
+import edu.uci.ics.textdb.api.common.IField;
+
+public class DoubleField implements IField{
+    
+    private Double value;

```

Comment:
If getValue() is really needed, I want to hear @JavierJia 's opinion on how to avoid this unnecessary object creation.